pyEpiabm/pyEpiabm/sweep/travel_sweep.py

```diff
@@ -0,0 +1,239 @@
+#
+# Introduce to and remove individuals from population due to travelling
+#
+import numpy as np
+import random
+import math
+
+from pyEpiabm.core import Population, Parameters, Microcell
+from pyEpiabm.property import InfectionStatus
+from pyEpiabm.sweep import HostProgressionSweep
+from .abstract_sweep import AbstractSweep
+
+
+class TravelSweep(AbstractSweep):
+    """Class to run the introduction and removing of
+    individuals. The number of introduced individuals
+    depends on the number of total infectious cases in the
+    population. All indivuals are infectious when entering
+    the population and will distributed over microcells based
+    on population density of the microcells. Individuals will
+    be removed from the population after a certain number of days
+    depening on if they are isolated or quarantined.
+
+    """
+
+    def __init__(self):
+        """Call in variables from the parameters file
+        """
+        self.travel_params = Parameters.instance().travel_params
+        self.introduce_population = Population()
+        self.introduce_population.add_cells(1)
+        self.initial_cell = self.introduce_population.cells[0]
+        self.initial_cell.add_microcells(1)
+        self.initial_microcell = self.initial_cell.microcells[0]
+
+    def __call__(self, time: float):
+        """ Based on number of infected cases in population, infected
+        individuals are introduced to the population for a certain
+        period. They are distributed over the microcells based on
+        population density. They are not assigned perminently to
+        a place for the durationg of their visit.
+
+        Parameters
+        ----------
+        time : float
+            Simulation time
+        """
+        # Introduce number of individuals
+        num_cases = sum(map(lambda cell: cell.number_infectious(),
+                        self._population.cells))
+        number_individuals_introduced = math.floor(
+            num_cases * self.travel_params['ratio_introduce_cases'])
+
+        if number_individuals_introduced >= 1:
+            self.create_introduced_idividuals(
+                time, number_individuals_introduced)
+            self.assign_microcell_household(
+                number_individuals_introduced)
+
+            # Remove introduced individuals from introduce population
+            self.initial_cell.persons = []
+            self.initial_microcell.persons = []
+
+        # Remove individuals if duration of stay passed
+        self.remove_leaving_individuals(time)
+
+    def create_introduced_idividuals(self, time,
```

Comment:
```suggestion
    def create_introduced_individuals(self, time,
```

You'll need to change this wherever it was called too

Reply:
Done